scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/package.scala

```diff
@@ -200,9 +200,17 @@ package object sparkey extends SparkeyReaderInstances {
     uri
   }
 
-  /** Enhanced version of [[com.spotify.scio.values.SCollection SCollection]] with Sparkey methods. */
-  implicit class SparkeyPairSCollection[K, V](@transient private val self: SCollection[(K, V)]) {
+  private object SparkeyPairSCollection {
     private val logger = LoggerFactory.getLogger(this.getClass)
+  }
+
+  /** Enhanced version of [[com.spotify.scio.values.SCollection SCollection]] with Sparkey methods. */
+  implicit class SparkeyPairSCollection[K, V](@transient private val self: SCollection[(K, V)])
```

Comment:
@nevillelyh since you are making changes here 😄 can you remove `@transient`? we don't really need it here